### DISCUSSION TITLE
org.eclipse.justj.openjdk.hotspot.jre.full error

### DISCUSSION THREAD

**Discussion by zyang946** [eclipse-justj/justj#14] [Q&A]
<feature id="org.eclipse.justj.openjdk.hotspot.jre.full" installMode="root"/> 
eclipse-plugin artifact with ID "org.eclipse.justj.openjdk.hotspot.jre.full" and version matching "0.0.0" was not found in the target platform

**Comment by merks** [comment]
This page documents all the update sites where you can find it

https://download.eclipse.org/justj/www/?page=download

E.g.,

https://download.eclipse.org/justj/jres/17/updates/release/latest
